Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py

```diff
@@ -1274,6 +1279,38 @@ def channel_user_list_command():
     return_results(result)
 
 
+def is_bot_in_chat(chat_id):
+    """
+    check if the bot is already in the chat.
+    """
+    url_suffix = f"v1.0/chats/{chat_id}/installedApps?$expand=teamsApp," \
+                 f"teamsAppDefinition&$filter=teamsApp/externalId eq '{BOT_ID}'"
+    res = http_request('GET', urljoin(GRAPH_BASE_URL, url_suffix))
+    return True if res.get('value') else False      # type: ignore
+
+
+def add_bot_to_chat(chat_id: str):  # pragma: no cover
+    """
+    Add the Dbot to a chat.
+    :param chat_id: chat id which to add the bot to.
+    """
+
+    demisto.debug(f'adding bot with id {BOT_ID} to chat')
+
+    # bot is already part of the chat
+    if is_bot_in_chat(chat_id):
+        return
+
+    res = http_request('GET', f"{GRAPH_BASE_URL}/v1.0/appCatalogs/teamsApps?$filter=externalId eq '{BOT_ID}'")
```

Comment:
Please use params instead of explicitly writing the params as part of the suffix.

Reply:
same as above